### DISCUSSION TITLE
Kali on UTM

### DISCUSSION THREAD

**Discussion by bhartleyy** [utmapp/UTM#5544] [Linux]
I've been trying to run Kali on UTM on my M1 Mac for quite a while now, I've ensured that I've got the correct iso for my architecture but I still keep hitting dead ends. Will attach pics.
![image](https://github.com/utmapp/UTM/assets/100302603/9cdb73d1-9459-4fa3-a465-49a52340cb95)

![image](https://github.com/utmapp/UTM/assets/100302603/77bbf067-7f88-478a-8b49-4ffdcaa9c133)

![image](https://github.com/utmapp/UTM/assets/100302603/48ff11bf-72e9-41dc-988b-5c2812be3232)

Mainly receive this error ^^

**Comment by happyhaerin** [comment]
This youtube link was helpful for me:
https://youtu.be/9zdjQ9w_v_4
